src/t8_netcdf.h

```diff
@@ -0,0 +1,84 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+/** file t8_netcdf.h
+ * This Header-File holds two typedef integer datatypes for the data of NetCDF integer variables, as well as a struct and functions to create extern elementwise data-variables which should be written additionally to a Forest or Cmesh NetCDF-File.
+ */
+
+#ifndef T8_NETCDF_H
+#define T8_NETCDF_H
+
+#include <t8.h>
+
+/* Datatype for 64 bit NetCDF Data */
+typedef int64_t     t8_nc_int64_t;
+
+/* Datatype for 32 bit NetCDF Data */
+typedef int32_t     t8_nc_int32_t;
+
+/* Struct for elementwise data variable for a NetCDF file */
+typedef struct
+{
+  const char         *variable_name;
+  const char         *variable_long_name;
+  const char         *variable_units;
+  int                 datatype;
```

Comment:
I think an enum would be more clear here. If you stay with int, there needs to be at least an explanation that 0=int, 1=double